kratos.gid/apps/DEM/xml/Parts.spd

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<condition n="Parts" pn="Parts" ov="point,line,surface,volume" ovm="element" icon="shells16" help="Select your group" un="DEMParts">
+<condition n="Parts" pn="Parts" ov="point,line,surface,volume" ovm="node,element" icon="shells16" help="Select your group" un="DEMParts">
```

Comment:
This is the important change of this branch
@farrufat-cimne 
@KratosMultiphysics/dem